discord/message.py

```diff
@@ -1991,6 +1997,17 @@ def to_message_reference_dict(self) -> MessageReferencePayload:
 
         return data
 
+    def to_dict(self) -> MessagePayload:
+        """Converts this message object into a dict.
```

Comment:
I wouldn't say this converts anything. Maybe it can be phrased differently ? Also what do others think of this ? Do we still want `.to_dict()` of there's `.data` ?

Reply:
thats kinda useless